config/environments/development.rb

```diff
@@ -28,8 +28,21 @@
     config.cache_store = :null_store
   end
 
+  # Tell Action Mailer to use smtp server
+  config.action_mailer.delivery_method = :smtp
+
+  ActionMailer::Base.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: "587",
+    domain: "gmail.com",
+    user_name: "greenlight.blindside@gmail.com",
+    password: "steponthegas",
```

Comment:
We should never include hardcoded configuration. All this has to be configured using ENV variables